proto/aggregation.proto

```diff
@@ -152,3 +139,16 @@ message Overflow {
   bytes overflow_service_transactions_estimator = 5;
   bytes overflow_spans_estimator = 6;
 }
+
+message Bar {
```

Comment:
`HDRHistogramBar`?

IIANM adding this message will increase the amount of pointer indirection, and memory pressure, compared to https://github.com/elastic/apm-aggregation/pull/49.

OTOH I like the approach of keeping the histogram sorted at all times. Did you already consider and investigate doing this with the existing pair of arrays representation?

Reply:
> Did you already consider and investigate doing this with the existing pair of arrays representation?

Good point, I didn't do it. I will use counts and buckets instead of `Bar` for the proto representation.

> OTOH I like the approach of keeping the histogram sorted at all times

Another advantage of this is that it makes the `Buckets()` calculation more efficient (which is why we see the improvement in `CombinedMetricsToBatch` w.r.t. `time/op`.

Reply:
Updated the code to remove `Bar` message from proto definition.